src/view/uniprotkb/components/UniProtCard.tsx

```diff
@@ -78,41 +85,43 @@ const UniProtCard: FC<{
     }
   }
 
-  let functionNode;
-  const firstCommentType = idx(data, (_): string => _.comments[0].commentType);
-  if (data.comments && firstCommentType === CommentType.FUNCTION) {
-    const firstComment = data.comments[0] as FreeTextComment;
-    const firstCommentValue = idx(
-      firstComment,
-      (_): string => _.texts[0].value
-    );
-    if (firstCommentValue) {
-      functionNode = truncate(firstCommentValue, {
-        length: CHAR_LENGTH_FUNCTION_SUMMARY,
-      });
-    }
-  }
-
   return (
-    <div>
-      <h3>
-        <UniProtTitle
-          primaryAccession={data.primaryAccession}
-          entryType={data.entryType}
-          uniProtId={data.uniProtId}
-        />
-      </h3>
-      <p>
-        {recommendedNameNode}
-        {organismNameNode}
-        {geneNameListNode}
-        {sequenceLengthNode}
-        {annotationScoreNode}
-      </p>
-      <p>{functionNode}</p>
-      <p>{keywordsNode}</p>
-    </div>
+    <Card
+      links={highlights}
+      onClick={() => history.push(`/uniprotkb/${data.primaryAccession}`)}
+    >
+      <section className="uniprot-card">
+        <section className="uniprot-card__left">
+          <input
+            type="checkbox"
+            checked={selectedEntries[data.primaryAccession]}
+            onClick={e => e.stopPropagation()}
+            onChange={() => handleEntrySelection(data.primaryAccession)}
+            data-testid="up-card-checkbox"
+          />
+        </section>
+        <section className="uniprot-card__right">
+          <h5>
+            <UniProtTitle
+              primaryAccession={data.primaryAccession}
+              entryType={data.entryType}
+              uniProtId={data.uniProtId}
+            />
+          </h5>
+          <section>
+            {recommendedNameNode}
+            {organismNameNode}
+            {geneNameListNode}
+            {sequenceLengthNode}
```

Comment:
The creation of `sequenceLengthNode`:
```
const sequenceLengthNode = `${data.sequence.length} amino-acids · `;
```
throws the following error:
```
Uncaught TypeError: Cannot read property 'length' of undefined
    at UniProtCard (UniProtCard.tsx:
```
when the results are loaded in the table view and then changed to the cards view.